src/main/host/process.rs

```diff
@@ -517,6 +517,10 @@ impl Process {
         self.itimer_real.borrow_mut()
     }
 
+    pub fn get_thread(&self, virtual_tid: ThreadId) -> Option<impl Deref<Target = ThreadRef> + '_> {
```

Comment:
Drop `get_` prefix?

Reply:
Actually, call this `thread_borrow`, since it borrows the thread list? (And can/will borrow a thread object in that list)

Reply:
Oops yeah that's better. Updated to `thread_borrow`.